src/neuvector/values/values.yaml

```diff
@@ -20,6 +20,8 @@ controller:
         no_telemetry_report: true
         scan_config:
           auto_scan: true
+        mode_auto_d2m: true
+        mode_auto_d2m_duration: 3600
```

Comment:
Do you think 24 hours might be a safer bet for workloads? Maybe overly cautious since it's just monitor mode but I'd suspect some of our workloads/end user workloads might do things on a schedule that is greater than 1 hour (loki indexing comes to mind, unsure if that would trigger different processes/network calls though).

Reply:
per our chat, we want to keep the number as low as possible while still trying to capture the various network traffic expected from uds-core pieces.  so we were thinking this probably ought to be at least 24-48 hours.  so planning to just split the difference, and go with 36 hours.